src/coreComponents/fileIO/vtk/VTKPVDWriter.cpp

```diff
@@ -52,5 +52,13 @@ void VTKPVDWriter::addData( real64 time, string const & filePath ) const
   dataSetNode.append_attribute( "timestep" ) = time;
   dataSetNode.append_attribute( "file" ) = filePath.c_str();
 }
+
+void VTKPVDWriter::reinitData() const
```

Comment:
Is there a way we can access `m_pvdFile` from `VTKOutput` and put these 2 lines directly in `VTKOuput::reinit()` ? This would avoid to have 3 reinit() methods ( `reinit()` `reinitData()` `clearData()` ).